src/components/join/JoinUs.tsx

```diff
@@ -4,7 +4,7 @@ import Image from "next/image";
 const JoinUs = () => {
   return (
     <div className="min-h-screen">
-      <div className="flex flex-col items-center justify-center pt-20">
+      <div className="flex flex-col items-center justify-center pt-20 not-visited:flex-col md:flex">
```

Comment:
```suggestion
      <div className="flex flex-col items-center justify-center pt-20">
```

these dont seem to affect the styling